### ISSUE TITLE
Website: Resource cards at top of component pages

### ISSUE THREAD

**Issue by janchild** [carbon-design-system/carbon-for-ibm-dotcom#6540]
## Task
- [ ] Edit the `ComponentFooter` and `SpecLinks` component to render the spec links as **two Resource cards**. 
- [ ] Place Resource cards at top of page, as 1st item under Anchor links 

Useful info: https://gatsby-theme-carbon.vercel.app/components/ResourceCard

### Content: 
H2 Heading: `Resources`

Card 1: 
- Card heading `Design specifications`
- Sketch icon
- link to box folder of component 

Card 2: 
- Card heading `Functional specifications`
- Github icon
- link to Website Github wiki

**Note:** Even though Resources is an H2 heading, it doesn't get an anchorlink. It's more of a handy feature (quick dive in) than part of the chronological structure of the page. This was a decision made with Mike Abbink for IDL and also the Carbon web pages. 

![Card group resource sample.png](https://images.zenhubusercontent.com/5ca77ffc7a18154b83385e9b/48baaa95-daf0-499c-ab1b-5890458fe3ed)

### Background

This is a "quick win" to make the design specs and functional specs easier to find. 

Following the model shown here: https://www.ibm.com/design/language/color

This change would be made across all components.

**Comment by shixiedesign** [comment]
It's a dev task so returning to To be triaged col for discussion

**Comment by janchild** [comment]
@ariellalgilmore, thanks for meeting this morning. Here is the mockup of what we talked about this morning. This addition would involve removing both the `Design and functional specificationss` and `Development documentation` sections from the page and the anchorlinks. And replacing everything contained in these sections with mini cards.

Just like before, the mini cards would be in the first section called `Resources` and this H2 would not be included in the anchorlinks.  

I decided to go with the React icon for the React card, to leverage that visual recognition. I'm not sure if there is an icon for Web Components, so I just put the Storybook one in there as a placeholder for now. 

Let me know if you have any questions! 

![sample resource cards](https://user-images.githubusercontent.com/44503588/128378217-56d8f92c-60de-4828-b778-df18114a310f.png)
